src/main/drivers/flash.c

```diff
@@ -128,7 +128,7 @@ static bool flashSpiInit(const flashConfig_t *flashConfig)
 #ifndef FLASH_SPI_SHARED
     //Maximum speed for standard READ command is 20mHz, other commands tolerate 25mHz
     //spiSetDivisor(busdev->busdev_u.spi.instance, SPI_CLOCK_FAST);
-    spiSetDivisor(busdev->busdev_u.spi.instance, SPI_CLOCK_STANDARD*2);
+    spiSetDivisor(busdev->busdev_u.spi.instance, SPI_CLOCK_FAST);
```

Comment:
This increases configured speed  8 times. Is that intentional?

Reply:
I checked the speeds against the data sheets for each device. As you say, these are big improvements in speed. 16x in some cases. I can only assume they were set slow to aid debug, and then left that way by mistake.